bin/peggy-cli.mjs

```diff
@@ -489,7 +497,49 @@ export class PeggyCLI extends Command {
     }
     if (typeof this.testText === "string") {
       this.verbose("TEST TEXT:", this.testText);
-      const exec = eval(source);
+
+      const filename = this.outputJS
+        ? path.resolve(this.outputJS)
+        : path.join(process.cwd(), "stdout.js"); // Synthetic
+
+      const dirname = path.dirname(filename);
+      const m = new Module(filename, module);
```

Comment:
I suppose this is fix for #211. It is better to add an explanation of why and how it works (a link to the receipt if you cannot describe that by your words)

Reply:
Added a comment.  Let me know if you want more.

Reply:
That's great!